### PR TITLE
Added Conda install instructions. Fixes issue #535.

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -674,6 +675,27 @@ then the configuration stops because of this difference.
...
+[Miniconda](http://conda.pydata.org/miniconda.html) or
+[Anaconda](https://www.continuum.io/downloads). Either of these will provide
+the `conda` command which can be invoked at the command line to install Simbody
+from the [Conda Forge](https://conda-forge.github.io/) channel as follows:
+
+```
+$ conda install -c conda-forge simbody
+```
+
+This command will install Simbody (both the shared objects and headers) into
```
Comment (chrisdembia) [inline README.md]: objects -> libraries

File: README.md [hunk 2]
```diff
@@ -117,13 +117,14 @@ Using Simbody
...
 
-Simbody works on Windows, Mac, and Linux. For Windows, you must build from source. For Mac and Linux, you can use a package manager or build from source. In this file, we provide instructions for 5 different ways of installing Simbody:
+Simbody works on Windows, Mac, and Linux. For each operating system, you can use a package manager or build from source. In this file, we provide instructions for 6 different ways of installing Simbody:
 
 1. [**Windows**](#windows-using-visual-studio): build from source using Microsoft Visual Studio.
 2. [**Linux or Mac (make)**](#linux-or-mac-using-make): build from source using gcc or Clang with make.
 3. [**Mac (Homebrew)**](#mac-and-homebrew): automated build/install with Homebrew.
 4. [**Ubuntu/Debian**](#ubuntu-and-apt-get): install pre-built binaries with apt-get.
 5. [**Windows using MinGW**](#windows-using-mingw): build from source using MinGW.
+6. [**Windows/Mac/Linux**](#windows-mac-and-linux-using-conda): install pre-built binaries with Conda.
```
Comment (chrisdembia) [inline README.md]: Consider replacing "Conda" with "the Conda package manager"

**Comment by moorepants** [comment]
Chris, I addressed the comments here, fyi.

**Comment by chrisdembia** [comment]
LGTM.

**Comment by sherm1** [comment]
Thanks @moorepants!

**Pull request merged**